src/common/meta/src/key.rs

```diff
@@ -36,6 +36,19 @@
 //!     - The value is a [TableNameValue] struct; it contains the table id.
 //!     - Used in the table name to table id lookup.
 //!
+//! 6. Flow task key: `__flow_task/{task_id}`
+//!     - Stores metadata of the task.
+//!
+//! 7. Flow task key: `__flow_task_name/{catalog}/{task}`
```

Comment:
```suggestion
//! 7. Flow task key: `__flow_task_name/{catalog}/{task_name}`
```

Reply:
Why is the flow task is directly put under "catalog"? Do you mean flow task can cross database but not catalog?